tests/cases/models/negative_intmodel.slint

```diff
@@ -0,0 +1,54 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+
+
+TestCase := Rectangle {
+    width: 100phx;
+    height: 100phx;
+
+    property <int> creation-count: 0;
+
+    property <length> test-height: preferred-height;
+
+    property <int> neg-count: -10;
+
+    VerticalLayout {
+        for _ in neg-count: Rectangle {
+            preferred-height: 10px;
+            init => {
+                creation-count += 1;
+            }
+        }
+    }
+
+}
+
+
+/*
+```cpp
+auto handle = TestCase::create();
+const TestCase &instance = *handle;
+
+assert_eq(instance.get_creation_count(), 0);
+assert_eq(instance.get_test_height(), 0.);
+assert_eq(instance.get_creation_count(), 0);
```

Comment:
The test would be even better if it changed the neg-count after that to more, and check that the creation count is then that value.

Reply:
Good idea, will do.